apps/mobile/app/screens/Authenticated/SettingScreen/components/SingleInfo.tsx

```diff
@@ -40,11 +40,14 @@ const SingleInfo: FC<Props> = ({ title, value, onPress, onDetectTimezone }) => {
 					<AntDesign name="right" size={24} color="#938FA4" />
 				</TouchableOpacity>
 			) : (
-				<TouchableOpacity style={styles.toggle} onPress={() => (onPress ? onPress() : {})}>
+				<TouchableOpacity
+					style={dark ? styles.toggleDark : styles.toggleLight}
+					onPress={() => (onPress ? onPress() : {})}
+				>
 					{dark ? (
-						<Image style={{}} source={require("../../../../../assets/icons/new/toogle-dark.png")} />
+						<Image style={{}} source={require("../../../../../assets/icons/new/toggle-dark.png")} />
 					) : (
-						<Image source={require("../../../../../assets/icons/new/toogle-light.png")} />
+						<Image source={require("../../../../../assets/icons/new/toggle-light.png")} />
```

Comment:
@desperado1802 I don't think we should use images for the toggle, we should implement actual toggle button logic, the same as we have in Web.

Reply:
Used the same logic we use in the web app
https://github.com/ever-co/ever-teams/assets/124465103/83c54fcb-6ee0-4422-8259-15c453265cec